### PR TITLE
[BACKLOG-28664] Added Map-R 6 CE and EE drivers

### PR DESCRIPTION
**Pull request by angel-ramoscardona**

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 1m with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 39s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: assemblies/client/pom.xml [hunk 1]
```diff
@@ -200,6 +200,12 @@
                   <version>${pentaho-hadoop-shims-emr521.version}</version>
                   <type>kar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.pentaho</groupId>
+                  <artifactId>pentaho-hadoop-shims-mapr60-kar</artifactId>
+                  <version>${pentaho-hadoop-shims-mapr60.version}</version>
```
Comment (jgminder) [inline assemblies/client/pom.xml]: Move the version to the parent pentaho-karaf-assembly pom.xml in the dependencyManagement section.

File: assemblies/pme/pom.xml [hunk 2]
```diff
@@ -153,6 +153,12 @@
                   <version>${pentaho-hadoop-shims-emr521.version}</version>
                   <type>kar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.pentaho</groupId>
+                  <artifactId>pentaho-hadoop-shims-mapr60-kar</artifactId>
+                  <version>${pentaho-hadoop-shims-mapr60.version}</version>
```
Comment (jgminder) [inline assemblies/pme/pom.xml]: Remove the version after putting it in the parent pom's main pentaho-karaf-assembly's dependencyManagement section.

File: assemblies/prd/pom.xml [hunk 3]
```diff
@@ -135,6 +135,12 @@
                   <version>${pentaho-hadoop-shims-emr521.version}</version>
                   <type>kar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.pentaho</groupId>
+                  <artifactId>pentaho-hadoop-shims-mapr60-kar</artifactId>
+                  <version>${pentaho-hadoop-shims-mapr60.version}</version>
```
Comment (jgminder) [inline assemblies/prd/pom.xml]: Remove the version after putting it in the parent pom's main pentaho-karaf-assembly's dependencyManagement section.

File: assemblies/server/pom.xml [hunk 4]
```diff
@@ -176,6 +176,12 @@
                   <version>${pentaho-hadoop-shims-emr521.version}</version>
                   <type>kar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.pentaho</groupId>
+                  <artifactId>pentaho-hadoop-shims-mapr60-kar</artifactId>
+                  <version>${pentaho-hadoop-shims-mapr60.version}</version>
```
Comment (jgminder) [inline assemblies/server/pom.xml]: Remove the version after putting it in the parent pom's main pentaho-karaf-assembly's dependencyManagement section.

**Comment by jgminder** [comment]
This shim isn't going to ship for 9.0 so closing the PR.

**Pull request closed without merging**